src/js/Table/TableCell.ts

```diff
@@ -1,11 +1,27 @@
+type TableCellInitParams = {
+  table: any
+  rowData: any
+  rowIndex: number
+  columnIndex: number
+  updateRowData: any
```

Comment:
- `table` should be typed `TableProps` (import from `index`)
- `rowData` should be typed `Record<string, any>`
- `updateRowData` should be typed `(rowIndex: number, data: Array<Record<string, any>>, rerender: boolean, trigger: boolean, fieldName: string) => void`